### ISSUE TITLE
Update Input Fields to Meet WCAG 2.1 AA Standards

### ISSUE THREAD

**Issue by cyokum** [Fastspot/bigtree-form-builder#12]
[WCAG 2.1 Success Criterion 1.3.5 Identify Input Purpose](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html) requires field type attributes and autocomplete attributes. Can we get these added onto advance field types? Additionally, there should be the ability to add autocomplete attributes to standard fields.

**Comment by timbuckingham** [comment]
I think the referenced commit above should satisfy the WCAG 2.1 success criterion for field types that are beyond the regular text input.

In an effort of keeping form builder simple, I don't think adding the ability for users to specify autocomplete intents would be beneficial in the long run. I'd rather add more sub-types to fields or add more very-similar fields that a user could drag and drop over (e.g. a Photo field) than rely on a user's knowledge of valid HTML autocomplete values :)
